src/server/service/kv.rs

```diff
@@ -2062,12 +2062,13 @@ txn_command_future!(future_prewrite, PrewriteRequest, PrewriteResponse, (v, resp
     if let Ok(v) = &v {
         resp.set_min_commit_ts(v.min_commit_ts.into_inner());
         resp.set_one_pc_commit_ts(v.one_pc_commit_ts.into_inner());
-        GLOBAL_TRACKERS.with_tracker(tracker, |tracker| {
-            tracker.write_scan_detail(resp.mut_exec_details_v2().mut_scan_detail_v2());
-            tracker.write_write_detail(resp.mut_exec_details_v2().mut_write_detail());
-        });
+
     }
     resp.set_errors(extract_key_errors(v.map(|v| v.locks)).into());
+    GLOBAL_TRACKERS.with_tracker(tracker, |tracker| {
```

Comment:
I think we can move this block into txn_command_future macro to avoid copy it too many times.

Reply:
good idea